.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -0,0 +1,69 @@
+name: ecosystem-ci trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  trigger-all:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/ecosystem-ci run')
```

Comment:
Should we limit who can trigger this? (e.g. team members, PR author)

Reply:
Yes, ecosystem ci is very intense. Would limit to team, not even PR author

Reply:
Maybe cli like arg too to be able to call single framework tests? eg `/ecosystem-ci svelte`

Reply:
I don't know if the complexity of limiting it to the team is going to be needed, given that only close contributors will know how this works. I think we could try to leave it open and see how people use it, if there is abuse, we could restrict it. Maybe we could avoid sending to Discord the runs that are triggered by `/ecosystem-ci` and instead answer back with a comment on the PR?

+1 for passing options to `/ecosystem-ci` (maybe in future PRs)

Reply:
We discussed this with @dominikg in Discord, and he strongly suggest going with a whitelist. I'm fine with it, I would prefer if there is a way to have a list that doesn't depend on giving access to the Vite repo though (ideally any framework author doing a PR to vite should be able to trigger it). But we can keep it between the team to start with, as there are many of us active on PRs daily.